pgml-dashboard/src/tests.rs

```diff
@@ -0,0 +1,67 @@
+use pgml_dashboard::Clusters;
+use pgml_dashboard::{index, migrate, paths};
+use rocket::fs::FileServer;
+use rocket::{Build, Rocket};
+
+async fn rocket() -> Rocket<Build> {
```

Comment:
You can move this function inside the `test` module because it's the only place where it's being used.

Reply:
Moved rocket() into tests.